packages/terra-clinical-result/src/proptypes/observationPropTypes.js

```diff
@@ -12,34 +11,21 @@ const observationPropShape = PropTypes.shape({
   /**
    *  Value and optional Unit of Measure for the Observation Result
    */
-  result: PropTypes.oneOfType([
-    valueQuantityPropShape,
-    valueStringPropShape,
-    valueNullPropShape,
-  ]),
+  result: valueQuantityPropType,
   /**
    * Enum for possible Result Interpretation values (also called `Clinical Severity` and `Normalcy`).
-   * One of `'CRITICAL'`, `'EXTREMEHIGH'`, `'EXTREMELOW'`, `'PANICHIGH'`, `'PANICLOW'`, `'VABNORMAL'`, `'POSITIVE'`, `'ABNORMAL'`, `'HIGH'`, `'LOW'`, `'NORMAL'`, `'NEUTRAL'`.
+   * One of `'critical'`, `'critical-high'`, `'critical-low'`, `'positive'`, `'abnormal'`, `'high'`, `'low'`, `'normal'`.
    */
   interpretation: interpretationPropType,
-  /**
-   *  Enum for possible Result Types.
-   * One of `'ALPHA'`, `'MULTIALPHA'`, `'FREETEXT'`, `'NUMERIC'`, `'BLOODPRESSURE'`, `'CALCULATION'`, `'DATE'`, `'DATETIME'`, `'TIME'`, `'PROVIDER'`.
-   */
-  type: resultTypePropType,
   /**
    * Enum for possible Result Statuses.
-   * One of `'registered'`, `'preliminary'`, `'final'`, `'amended'`, `'corrected'`, `'cancelled'`, `'entered-in-error'`, `'unknown'`.
+   * One of `'final'`, `'entered-in-error'`s.
    */
   status: statusPropType,
   /**
-   *  Clinical datetime for the Result
-   */
-  performedDateTime: PropTypes.string,
-  /**
-   *  Last updated datetime for the Result
+   *  If the Result type is Numeric, in flowsheet cell will switch to alternate view if cannot fully display value.
    */
-  updateDateTime: PropTypes.string,
+  isTypeNumeric: PropTypes.bool,
```

Comment:
```suggestion
  isNumericResult: PropTypes.bool,
```

Reply:
Or even just `isNumeric`

Reply:
I'm fine either way - intention for keeping it with the "type" concept in the name because even blood pressures are numeric, but we don't want them to flip their presentation, didn't know if it would be misleading or overly tempting just if it was `isNumeric`. 

Key thing this does (it needs a new doc page just for special concepts like Numeric Type and and Statuses, Uncharted/In-error, and No Data) is this prop changes a very specific type of result to have this extra UI detail, mainly long numeric lab values, so we need to clearly indicate it's usage if we can.

<img width="744" alt="Screen Shot 2020-03-06 at 10 56 08 AM" src="https://user-images.githubusercontent.com/1355485/76104552-314d3580-5f99-11ea-98b0-7916da9efd5d.png">
.

Reply:
lets change this